js/ui/overlay.d.ts

```diff
@@ -83,6 +83,18 @@ export interface dxOverlayOptions<TComponent> extends WidgetOptions<TComponent>
      * @public
      */
     hideOnParentScroll?: boolean;
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    preventScrollEvents?: boolean;
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    enableBodyScroll?: boolean;
```

Comment:
Don't we wanna deprecate it to warn the users that we don't wanna keep old behavior in the future?

Reply:
No, we don't want to mark this option as deprecated in the future. It's a feature to provide capability to prevent body scrolling if customers are going to use Popup as Modal.

Reply:
Yeap, sorry, misschose a place - it was about `preventScrollEvents`. Even if we don't publish it, i think it's a good idea to raise a warning in console